src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py

```diff
@@ -827,15 +830,26 @@ def _get_cross_pollinate_fuzzers(engine_name, current_fuzzer_name):
                                              engine_name)
 
     cross_pollinate_fuzzers.append(
-        CrossPollinateFuzzer(
-            target,
-            backup_bucket_name,
-            corpus_engine_name,
+        uworker_msg_pb2.CrossPollinateFuzzerProto(
+            fuzz_target=uworker_io.entity_to_protobuf(target),
+            backup_bucket_name=backup_bucket_name,
+            corpus_engine_name=corpus_engine_name,
         ))
 
   return cross_pollinate_fuzzers
 
 
+def _get_cross_pollinate_fuzzers_from_protos(cross_pollinate_fuzzers_protos):
+  return [
+      CrossPollinateFuzzer(
```

Comment:
Can we replace this class with a proto? WDYT?

Reply:
The reason I went into this trouble is to keep compatibility with the untrusted runner version of this job:
https://github.com/google/clusterfuzz/blob/4ab81b3bbc1f168385a6d538862c1088f93616d8/src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py#L579

https://github.com/google/clusterfuzz/blob/4ab81b3bbc1f168385a6d538862c1088f93616d8/src/clusterfuzz/_internal/bot/untrusted_runner/tasks_host.py#L42

I am not sure if we want to keep this implementation or not.

Reply:
ACK. Yeah we need to keep this for now.